crates/node/builder/src/builder/mod.rs

```diff
@@ -677,20 +678,26 @@ impl<Node: FullNodeTypes> BuilderContext<Node> {
             + 'static,
         Node::Provider: BlockReaderFor<N>,
     {
-        self.start_network_with(builder, pool, Default::default())
+        self.start_network_with(
+            builder,
+            pool,
+            self.config().network.transactions_manager_config(),
```

Comment:
this was a drive by bug fix, because we currently didn't use the configured settings